sass/directives/_accents.scss

```diff
@@ -47,3 +47,54 @@
   width: 100%;
   left: 0;
 }
+
```

Comment:
I don't believe I'd consider this component an `accent` - how about creating a new file?